IBF-Typhoon-model/setup.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is there a reason why you want to use setup.py instead of just running the main script? It will complicate the development workflow a bit (see my comments on the dockerfile)